appium/sample-scripts/python/testdroid_android_hybrid.py

```diff
@@ -51,29 +59,25 @@ def setUp(self):
             while testdroid_device == "":
                 testdroid_device = deviceFinder.available_free_android_device()
 
-        
+
         desired_capabilities_cloud = {}
         desired_capabilities_cloud['testdroid_apiKey'] = testdroid_apiKey
-        desired_capabilities_cloud['testdroid_target'] = 'chrome'
-        desired_capabilities_cloud['testdroid_project'] = 'Appium Android Hybrid Demo'
-        desired_capabilities_cloud['testdroid_testrun'] = 'TestRun 1'
+        desired_capabilities_cloud['testdroid_target'] = 'selendroid'
```

Comment:
"selendroid" should be used only on API level < 17 and otherwise "android" ... one way to determine device API level is to ask from our API what Android version is the device at.